ember.s

```diff
@@ -196,26 +193,37 @@ not_rw:
 	jnz short not_print
 
 	mov cx, 16
-	mov si, di
+	xchg si, di
 	mov ah, 0x0e
 	mov bx, 0x0007
 .loop:	lodsb
 	int 0x10
 	loop .loop
 
+	xchg si, di
+
 	mov al, 0x0d
 	int i_putchar
 
 	jmp shell
 
 not_print:
 	cmp al, 'd'
-	jnz short parse_error ; TODO: make this a direct jump when features are finalized
+	jnz short not_drive
 	xchg ax, di
 	mov byte [do_disk.disknum+1], al
 
 	jmp shell
 
+not_drive:
+	cmp al, ';'
+	jnz short parse_error ; TODO: make this a direct jump when features are finalized
```

Comment:
This comment doesn't seem to be relevant anymore, since it referred to the unconditional jump to `parse_error` that was directly after a label before.